app/src/main/kotlin/com/hyperion/ui/screen/PlayerScreen.kt

```diff
@@ -173,21 +173,18 @@ private fun PlayerScreenLoaded(
                 ) {
                     items(viewModel.video!!.streams.filterIsInstance<DomainStream.Video>()) { stream ->
                         Row(
-                            modifier = Modifier.clickable { selectedStream = stream },
+                            modifier = Modifier.fillMaxWidth(1f).clickable { selectedStream = stream },
```

Comment:
This should be chained onto multiple new lines

Reply:
fixd, forc pushd